### PR TITLE
{bio}[foss/2019b] CAVIAR 2.2-20190419

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/5cfa888cc363e62d61065425c6ca6f2b for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/67725540de481ac66e446ee170fcaca1 for a full test report.

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/CAVIAR/CAVIAR-20200820-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+homepage = "https://github.com/fhormoz/caviar"
+description = """CAusal Variants Identication in Associated Regions. A statistical framework
+ that quantifies the probability of each variant to be causal while allowing an arbitrary
+ number of causal variants."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+sources = [{
+    'filename': 'CAVIAR-20200820.tar.gz',
+    'git_config': {
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/c/CAVIAR/CAVIAR-20200820-foss-2019b.eb]: To be more specific:

```python
local_commit_id = '135b58b'
version = '20190419'  # datestamp of commit
...
sources = [{
    'source_urls': ['https://github.com/fhormoz/caviar/archive/']
    'download_filename': '%s.tar.gz' % local_commit_id,
    'filename': SOURCE_TAR_GZ,
}]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node254.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2683 v4 @ 2.10GHz (broadwell), Python 2.7.5
See https://gist.github.com/eff577bb72cf7aedc55204cacfcf7552 for a full test report.

**Comment by smoors** [comment]
@verdurin here is the error:
```
g++ multipop.cpp PostCalMultiPop.cpp Util.cpp    -I /tmp/3349278.master01.hydra.brussel.vsc/tmp/vsc10009/build/CAVIAR/2.2-20190419/foss-2019b/caviar-135b58baffac92b5e9b45f8db78315a9b4d713bc/CAVIAR-C++/armadillo/include/ -DARMA_DONT_USE_WRAPPER -lopenblas -lgslcblas  -lgsl -o mupCAVIAR
Util.cpp:10:10: fatal error: gsl/gsl_linalg.h: No such file or directory
 #include <gsl/gsl_linalg.h>
          ^~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [mupCAVIAR] Error 1 
make: *** Waiting for unfinished jobs....
Util.cpp:10:10: fatal error: gsl/gsl_linalg.h: No such file or directory
 #include <gsl/gsl_linalg.h>
          ^~~~~~~~~~~~~~~~~~
Util.cpp:10:10: fatal error: gsl/gsl_linalg.h: No such file or directory
 #include <gsl/gsl_linalg.h>
          ^~~~~~~~~~~~~~~~~~
compilation terminated.
compilation terminated.
make: *** [eCAVIAR] Error 1
make: *** [setCAVIAR] Error 1
Util.cpp:10:10: fatal error: gsl/gsl_linalg.h: No such file or directory
 #include <gsl/gsl_linalg.h>
          ^~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [CAVIAR] Error 1 
```

**Comment by smoors** [comment]
@verdurin adding this line fixes it for me:
```
buildopts = 'CPATH=$EBROOTGSL/include:$CPATH'
```

**Comment by verdurin** [comment]
> buildopts = 'CPATH=$EBROOTGSL/include:$CPATH'

@smoors added that even though I couldn't reproduce the error locally.

File: easybuild/easyconfigs/c/CAVIAR/CAVIAR-20200820-foss-2019b.eb (hunk 1 above)
[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/c/CAVIAR/CAVIAR-20200820-foss-2019b.eb]: Done.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/42397e7d9c914b6f9a043d9534901c93 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/5316544045e4a49d1eb26ac4582bf69c for a full test report.

**Comment by smoors** [comment]
@boegelbot please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @verdurin!

**Pull request merged**